docs/creating-service-bindings/service-binding-resource.md

```diff
@@ -4,8 +4,122 @@ sidebar_position: 6
 
 # Service Binding Resource
 
-_In this section, we would highlight the different options based on the specification_
+## Service Binding Resource
 
-https://github.com/k8s-service-bindings/spec#service-binding
+Service Binding resources use `binding.operators.coreos.com` group with  `v1alpha1` version and `ServiceBinding` kind.
 
-_ We would refer to the specifcation about the Resource Type Schema_
\ No newline at end of file
+A sample Service Binding resource that connects a nodejs application to a postgresql database:
+
+
+```yaml
+apiVersion: binding.operators.coreos.com/v1alpha1
+kind: ServiceBinding
+
+metadata:
+  name: binding-request 
```

Comment:
I would chose some nicer name that reflects the intended consumption, e.g. 'db-binding'